app/assets/stylesheets/pages/_show.scss

```diff
@@ -48,4 +43,54 @@ h3 {
 
 ul {
   list-style-type: none;
+}
+
+.masonry-column {
+  padding: 0 1px;
+}
+
+.masonry-column div {
+  margin: 5px 1px;
+}
+
+.garden-image {
+  border-radius: 24px;
+  width: 100%;
+  object-fit: cover;
+}
+
+.garden-image-2 {
+  border-radius: 24px;
+  width: 50%;
+  object-fit: cover;
+}
+
+.feature i {
+  font-size: 32px;
+  color: $green;
+}
+
+.feature p {
+  font-size: 18px;
+  font-weight: bolder;
+  margin-bottom: 0px;
+}
+
+.feature span {
+  font-size: 16px;
+  font-weight: lighter;
+}
+
+#strike {
+  text-decoration: line-through;
```

Comment:
can we change this to just make the feature colour and font grey and maybe a lighter opacity?